docs/program/apis/disconnect.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "Session Connectivity: Network Disconnection during a Session"
+linkTitle: "Network Disconnection"
+weight: 20
+type: "docs"
+description: "How Viam handles losing connection to the internet during a robot session."
+tags: ["client", "sdk", "viam-server", "networking", "apis", "robot api", "session"]
+---
+
+What happens when your robot loses its connection to the internet and can no longer talk to the Viam cloud through [the Viam app](https://app.viam.com).
+
+### How does `viam-server` work if you lose connectivity during a session?
+
+If you lose connectivity, `viam-server` will by default timeout all resource and end this robot's [*session*](/program/apis/sessions/). 
+This does not immediately stop the flow of power to all actuating resources, but it signals that the robot can no longer receive operations over the internet, so any further operations will not be performed.
+
+This is because the robot cannot access Viam's APIs, including the robot API, without a connection to the cloud.
```

Comment:
```suggestion
This is because the robot cannot access Viam's APIs, including the robot API, without a connection to the cloud, and receive any more requests for operations to be performed.
```
Ok-- I might not get it, but is this better?

Reply:
No, I was more suggesting remove this line entirely.

Reply:
@Otterverse Ok that's good w me, removed